src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -118,7 +117,8 @@ void fillInnerParts() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
         personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
 
-        assignmentListPanel = new AssignmentListPanel(logic.getFilteredAssignmentList(Index.fromZeroBased(0)));
+        //TODO:
```

Comment:
Do you still need this ```//TODO:```?

Reply:
Yes, as I am planning to work more on the UI before 1.2b

Reply:
Alright!